spec/features/sign_in_with_valid_credentials_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature 'Signing in' do
+  scenario 'with valid credentials' do
+    sign_in_with_valid_credentials
+    expect(page).to have_content 'Welcome, JohnDoe!'
```

Comment:
JohnDoe is a magic string - you could have your sign_in method accept arguments for name, password or whatever, in order to make it clear that you are expecting the users name to show on the page.